### ISSUE TITLE
Support build parameters

### ISSUE THREAD

**Issue by kohsuke** [jenkinsci/jenkinsfile-runner#13]

**Comment by ndeloof** [comment]
I suggest we move to an option based CLI (`-x value`) so we gain more flexibility on usage

```
jenkinsfile-runner 
-w (--workdir) : working directory, default to "." 
-f (--file) : Jenkinsfile script, default to "Jenkinsfile
-p (--plugins) : plugins directory, default to $j/plugins
-j (--jenkins) : path to jenkins.war or exploded jenkins.war directory 
-c (--configuration) : path to a configuration-as-code file, default to $w/jenkins.yaml 
-a (--args) : build arguments
```
etc

**Comment by oleg-nenashev** [comment]
#26 was released in 1.0-beta-7 . I believe it resolves this feature request
